config/300-task.yaml

```diff
@@ -43,6 +43,18 @@ spec:
     # starts to increment
     subresources:
       status: {}
+  - name: v1
+    served: false
+    storage: false
+    schema:
+      openAPIV3Schema:
+        type: object
```

Comment:
Wondering if we might need the comments block from v1beta1 in #L34-L40, just in case when we remove v1beta1, this context is available in the v1 entry?
```
        # One can use x-kubernetes-preserve-unknown-fields: true
        # at the root of the schema (and inside any properties, additionalProperties)
        # to get the traditional CRD behaviour that nothing is pruned, despite
        # setting spec.preserveUnknownProperties: false.
        #
        # See https://kubernetes.io/blog/2019/06/20/crd-structural-schema/
        # See issue: https://github.com/knative/serving/issues/912
```

Reply:
I added a small explanation that is more up to date.